tracee-ebpf/tracee/net_events.go

```diff
@@ -30,6 +133,22 @@ func (t *Tracee) processNetEvents() {
 			// timeStamp is nanoseconds since system boot time
 			timeStampObj := time.Unix(0, int64(timeStamp+t.bootTime))
 
+			networkThread, networkThreadErr := t.getProcessCtx(hostTid)
+			if networkThreadErr != nil {
+				continue
+			}
+
+			// get process start time
+			procStartTime := networkThread.StartTime
+			if networkThread.HostTid != networkThread.HostPid {
+				networkProcess, err := t.getProcessCtx(networkThread.HostPid)
+				if err == nil {
+					procStartTime = networkProcess.StartTime
+				}
+			}
```

Comment:
Instead of performing this lookup for every packet, which is inefficient, we can keep the process start time in the process context of the thread

Reply:
agreed - i'll add it to the context.